src/templates/stickyboard-simple/src/components/base/PageBase.js

```diff
@@ -107,38 +112,39 @@ class PageBase extends React.Component {
     };
 
     handleInsert = () => {
-        const { layout, blocks } = this.state;
-
-        this.setState({
+        const {layout, blocks, EmptyStickerId} = this.state;
+        this.setState(
+            {
+            EmptyStickerId: uuid(),
```

Comment:
Please remove EmptyStickerId variable, and generate a new UUID every time when an empty sticker inserted.